tracker/settings/base.py

```diff
@@ -152,15 +151,6 @@
 WSGI_APPLICATION = 'tracker.wsgi.application'
 
 
-# Database
-# https://docs.djangoproject.com/en/1.10/ref/settings/#databases
-
-# Set the url as DATABASE_URL in the environment
-DATABASES = {
-    'default': dj_database_url.config(default="sqlite:///db.sqlite3")
-}
-
-
```

Comment:
RIP `dj_database_url`. I always thought this was an elegant way of specifying database connections, but I leave such decisions to the handlers of The Env Vars.